util/fpga/splice_nexysvideo.sh

```diff
@@ -16,20 +16,21 @@ set -e
 
 . util/build_consts.sh
 
-TARGET_PREFIX="sw/device/boot_rom/boot_rom_fpga_nexysvideo"
+TARGET_PREFIX="boot_rom/boot_rom_fpga_nexysvideo"
 TARGET="${DEV_BIN_DIR}/${TARGET_PREFIX}"
-FPGA_BUILD_DIR=build/lowrisc_systems_top_earlgrey_nexysvideo_0.1/synth-vivado/
+FPGA_BUILD_DIR=build/lowrisc_systems_top_earlgrey_nexysvideo_0.1/synth-vivado
+FPGA_MMI_PATH=${FPGA_BUILD_DIR}/lowrisc_systems_top_earlgrey_nexysvideo_0.1.runs/impl_1
```

Comment:
See the comment above regarding the file location. You should be able to remove have FPGA_BUILD_DIR == FPGA_MMI_PATH.

Reply:
Done